ark/utils/load_utils.py

```diff
@@ -172,7 +172,7 @@ def load_imgs_from_tree(data_dir, img_sub_folder=None, fovs=None, channels=None,
                           dims=["fovs", "rows", "cols", "channels"])
 
     # sort by fovs and channels for deterministic result
-    img_xr = img_xr.sortby('fovs').sortby('channels')
+    img_xr = img_xr.sortby('fovs')
```

Comment:
Same thing for FOVs: I think we should sort once we list all of the FOVs in the folder if None is provided. If a custom FOV list is provided, then we shouldn't sort. If order matters for some reason, I think it makes sense to keep it in the order that the items were provided in the function call.

Reply:
I think @ngreenwald was suggesting to remove this line. fovs and channels are both already sorted if they are `None`, and this line always sorts by fovs, even with user specified order.

Reply:
Yeah, forgot to do that. Will do for the next commit.